### PR TITLE
feat(pkglint): Make sure .snk files are ignored

### PR DESCRIPTION
**Pull request by RomainMuller**
Adds a `pkglint` rule to enforce presence of the `*.snk` pattern in both the `.gitignore`
and `.npmignore` file as a way to make it harder for one to accidentially push the
material to public locations.

This was missed in #643.

### CODE REVIEW

File: packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts [hunk 1]
```diff
@@ -9,6 +9,135 @@ import codebuild = require('../lib');
 
 export = {
   'default properties': {
+    'with build image'(test: Test) {
```
Comment (eladb) [inline packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts]: How is this related?

File: packages/@aws-cdk/aws-cloudformation/package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@aws-cdk/assert": "^0.14.1",
-    "@types/lodash": "^4.14.116",
+    "@types/lodash": "^4.14.117",
```
Comment (eladb) [inline packages/@aws-cdk/aws-cloudformation/package.json]: Intentional?

File: packages/@aws-cdk/cfnspec/package.json [hunk 3]
```diff
@@ -19,11 +19,11 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "devDependencies": {
-    "@types/fs-extra": "^4.0.8",
+    "@types/fs-extra": "^5.0.4",
```
Comment (eladb) [inline packages/@aws-cdk/cfnspec/package.json]: Don't mix this PR with anything else...

File: packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts (hunk 1 above)
Comment (RomainMuller) [reply packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts]: How did this get in? I didn't even git add this change o_O

File: packages/@aws-cdk/aws-cloudformation/package.json (hunk 2 above)
Comment (RomainMuller) [reply packages/@aws-cdk/aws-cloudformation/package.json]: Yes. I forgot to mention it in the PR description... #oops

File: packages/@aws-cdk/cfnspec/package.json (hunk 3 above)
Comment (RomainMuller) [reply packages/@aws-cdk/cfnspec/package.json]: Yeah - made a force-push with just the new filter.

**Comment by eladb** [comment]
Please reference the offending original PR.
